packages/medusa/src/services/sales-channel.ts

```diff
@@ -161,10 +161,23 @@ class SalesChannelService extends TransactionBaseService<SalesChannelService> {
         () => void 0
       )
 
+      // if sales channel does not exist, return early
```

Comment:
comment: not sure that comment is necessary since I think that the code is self descrptive here. wdyt?